### DISCUSSION TITLE
Add slider for "FINAL_RATE_PROGRESS" in ModTimeRamp

### DISCUSSION THREAD

**Discussion by Panthr75** [ppy/osu#15291] [Ideas]
I thought it was always jank that a song you were playing with Wind Up would be at maximum speed before you ended the level, and having a slider for it would be more intuitive to the user.

**Comment by Panthr75** [comment]
Example:
![image](https://user-images.githubusercontent.com/29237603/138706990-b48043b4-8d4e-469f-b720-696ed3d15d81.png)

**Comment by peppy** [comment]
If you think it's "jank", it's probably worth reassessing the defaults. Maybe it should be changed rather than a setting be added.

**Reply by Panthr75** [reply]
I would consider it, but there may be people who like the 75% of the level having 100% of the final speed. I just am not one of those people, and hey, it allows for more customization in the long run.

**Reply by peppy** [reply]
Well I agree with changing the default personally, also think it feels weird that it stops.
